packages/web/components/swap-tool/index.tsx

```diff
@@ -667,46 +673,42 @@ export const SwapTool: FunctionComponent<SwapToolProps> = observer(
                     setShowEstimateDetails((show) => !show);
                 }}
               >
-                <span
-                  className={classNames("subtitle2 transition-opacity", {
-                    "text-osmoverse-600": !isQuoteDetailRelevant,
-                    "opacity-50":
-                      swapState.isQuoteLoading ||
-                      swapState.inAmountInput.isTyping,
-                  })}
-                >
-                  1{" "}
-                  <span title={swapState.fromAsset?.coinDenom}>
-                    {ellipsisText(
-                      swapState.fromAsset?.coinDenom ?? "",
-                      isMobile ? 11 : 20
-                    )}
-                  </span>{" "}
-                  {`≈ ${
-                    swapState.toAsset
-                      ? formatPretty(
-                          (swapState.quote?.inOutSpotPrice
-                            ? new CoinPretty(
-                                swapState.toAsset,
-                                swapState.quote.inOutSpotPrice.mul(
-                                  DecUtils.getTenExponentN(
-                                    swapState.toAsset.coinDecimals
-                                  )
-                                )
-                              )
-                            : null) ??
-                            swapState.spotPriceQuote?.amount ??
-                            new Dec(0),
-                          {
-                            maxDecimals: Math.min(
-                              swapState.toAsset.coinDecimals,
-                              8
-                            ),
-                          }
-                        )
-                      : "0"
-                  }`}
-                </span>
+                {swapState.toAsset && isSwapAssetsSpotPriceAvailable && (
```

Comment:
Made it so that the spot price is hidden if it is unavailable from quote or loading instead of being zero